x/commerciodocs/keeper.go

```diff
@@ -34,14 +34,14 @@ type Keeper struct {
 }
 
 func NewKeeper(
-	commercioIdKeeper commercioid.Keeper,
+	commercioId commercioid.Keeper,
```

Comment:
Can we revert this renaming and use the old `commercioIdKeeper` instead of `commercioId`?
 I think it might be later more clear using the first one